main/station-config.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const Store = require('electron-store')
+const configStore = new Store()
+
+const ConfigKeys = {
+  // FilAddress: 'station.filAddress',
+  OnboardingCompleted: 'station.sawOnborading',
+  UserConsent: 'station.userConsent'
+}
+
+// let filAddress = /** @type {string | undefined} */ (configStore.get(ConfigKeys.FilAddress))
+let OnboardingCompleted = /** @type {boolean | undefined} */ (configStore.get(ConfigKeys.OnboardingCompleted))
+let userConsent = /** @type {boolean | undefined} */ (configStore.get(ConfigKeys.UserConsent))
+
+// /**
+//  * @returns {string | undefined}
+//  */
+//  function getFilAddress () {
+//   return filAddress
+// }
+
+// /**
+//  * @param {string | undefined} address
+//  */
+// function setFilAddress (address) {
+//   filAddress = address
+//   configStore.set(ConfigKeys.FilAddress, address)
+// }
```

Comment:
IMO, we should eventually move the implementation of `saturnNode.setFilAddress` & `saturnNode.getFilAddress` here. As part of that change, it would be great to implement migration and lift the FIL address configured for Saturn to become the FIL address configured for the entire Station. I think such work is out of the scope of this pull request.

I am proposing to either remove this commented-out block of code, or add a comment to explain what is our plan.